niworkflows/anat/ants.py

```diff
@@ -0,0 +1,559 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""
+Nipype translation of ANTs workflows
+------------------------------------
+
+"""
+from __future__ import print_function, division, absolute_import, unicode_literals
+
+# general purpose
+import os
+from multiprocessing import cpu_count
+from pkg_resources import resource_filename as pkgr_fn
+from packaging.version import parse as parseversion, Version
+from pathlib import Path
+
+# nipype
+from nipype.pipeline import engine as pe
+from nipype.interfaces import utility as niu
+from nipype.interfaces.fsl.maths import ApplyMask
+from nipype.interfaces.ants import N4BiasFieldCorrection, Atropos, MultiplyImages
+
+# niworkflows
+from ..data import TEMPLATE_MAP, get_dataset
+from ..interfaces.ants import (
+    ImageMath,
+    ResampleImageBySpacing,
+    AI,
+    ThresholdImage,
+)
+from ..interfaces.fixes import (
+    FixHeaderRegistration as Registration,
+    FixHeaderApplyTransforms as ApplyTransforms,
+)
+
+ATROPOS_MODELS = {
+    'T1': (3, 1, 2, 3),
+    'T2': (3, 3, 2, 1),
+    'FLAIR': (3, 1, 3, 2),
+}
```

Comment:
This is opaque. At the least I would specify that this is `(K, CSF, GM, WM)`, but we could also do something a bit more structured, like:

```Python
ATROPOS_MODELS = {
   'T1': {'wm': 3, 'gm': 2, 'csf': 1},
   'T2': {'wm': 1, 'gm': 2, 'csf': 3},
   'FLAIR': {'wm': 2, 'gm': 3, 'csf': 1},
}
```

Or:

```Python
ATROPOS_MODELS = {
   'T1': ['csf', 'gm', 'wm'],
   'T2': ['wm', 'gm', 'csf'],
   'FLAIR': ['csf', 'wm', 'gm'],
}
```

And then we can end up with something a little more clear in the `_select_labels`, as well. Just a thought, and maybe one for another day.

Reply:
I've switched to OrderedDict for the specification, but I'm +1 in using plain dicts and modifying `_select_labels`